### ISSUE TITLE
About the training loss

### ISSUE THREAD

**Issue by wmyw96** [duxingren14/DualGAN#22]
I read your code about the design for loss and found your implementation is different from that proposed in the paper. So you use the traditional loss of GAN instead of the WGAN loss? Does it mean WGAN loss might not be a good choice in practice?

**Comment by duxingren14** [comment]
The result images shown in the main body of the paper are generated with
WGAN. After the acceptance of the paper, I did some further experiments and
found that the original GAN loss actually help improves the quality of
output images, for the datasets used in the paper. Though I cannot conclude
that the traditional GAN loss is always better than wGAN loss in terms of
the DualGAN architecture.

If you have interests in further exploration, you may try WGAN-GP or other
losses.

**Comment by chl916185** [comment]
Why is noise z not used?
@duxingren14

**Comment by duxingren14** [comment]
Instead of explicitly adding random noise, I used dropout instead.
